spec/duration.html

```diff
@@ -102,19 +102,23 @@ <h1>Temporal.Duration.compare ( _one_, _two_ [ , _options_ ] )</h1>
         1. Let _calendarUnitsPresent_ be *false*.
         1. If any of _one_.[[Years]], _two_.[[Years]], _one_.[[Months]], _two_.[[Months]], _one_.[[Weeks]], or _two_.[[Weeks]] are not 0, then
           1. Set _calendarUnitsPresent_ to *true*.
+        1. Let _calendarRec_ be *undefined*.
+        1. If _relativeTo_ is not *undefined*, then
+          1. Set _calendarRec_ to ! CreateCalendarRecord(_relativeTo_.[[Calendar]], « »).
+          1. If _calendarUnitsPresent_ is *true*, then
+            1. Set _calendarRec_.[[DateAdd]] to ? GetMethod(_relativeTo_.[[Calendar]], *"dateAdd"*).
```

Comment:
Note to self: when rebasing, as per #2547, need to handle lookups on string calendar slot values. Probably the best way is to introduce a `? CalendarRecordLookup(_calendarRec_, ~dateAdd~)` operation.